great_expectations/data_context/data_context.py

```diff
@@ -1634,6 +1634,9 @@ def get_validator(
         data_asset_name: Optional[str] = None,
         *,
         batch_request: Optional[Union[BatchRequest, RuntimeBatchRequest]] = None,
+        batch_request_list: List[
```

Comment:
@talagluck FWIW: I still do not understand the purpose of `List[BatchRequest]` instead of using `get_batch_list()` multiple times with one `BatchRequest` at a time.  If this is designed in order to populate the loaded batches in `Validator` with all batches to be identifiable by all uses of `BatchRequest` in a `data_context` session (pipeline/notebook, Checkpoint, ParameterBuilder), then I would like to keep talking about it.  Thank you.

Reply:
Hi @alexsherstinsky! 

This was a requested feature by James from the beginning of this process (see GE-24), and we confirmed yesterday in the design review that this was something that he definitely wanted. 

If we are allowing the loading of multiple BatchRequests on a Validator (which we are doing with PR #2786), then enabling this loading with multiple BatchRequests at one time is a logical step in conjunction. If a user knows all of the BatchRequests they want on a Validator at the start then it seems like a strange pattern to require them to select just one of the BatchRequests with which to instantiate the Validator, then load additional BatchRequests, when instantiating in this way is straightforward to implement and has the same end result.

Reply:
@talagluck Thank you for the clarification.  I am approving this now.  Having said that, I think that the opposite usage pattern would be more user-friendly.  In other words, letting users issue one `BatchRequest` at a time is easier than knowing all ahead of time.  There are analogs in other products where requiring programmers to plan ahead was later revised into a more of an "as-you-go" model.  Adding @jcampbell so that we can perhaps later reconsider (once we gain more experience with these APIs).  Thank you!